webkitgtk-1.3.13 and up require pthread functions not yet available in Cygwin

--- origsrc/epiphany-3.0.0/configure.ac	2011-04-14 19:34:16.651811400 -0500
+++ src/epiphany-3.0.0/configure.ac	2011-04-14 19:34:27.384630100 -0500
@@ -93,7 +93,7 @@ LIBXML_REQUIRED=2.6.12
 LIBXSLT_REQUIRED=1.1.7
 LIBSTARTUP_NOTIFICATION_REQUIRED=0.5
 DBUS_GLIB_REQUIRED=0.35
-WEBKIT_GTK_REQUIRED=1.4.2
+WEBKIT_GTK_REQUIRED=1.3.12
 LIBSOUP_GNOME_REQUIRED=2.33.1
 GNOME_KEYRING_REQUIRED=2.26.0
 GSETTINGS_DESKTOP_SCHEMAS_REQUIRED=0.0.1
--- origsrc/epiphany-3.0.0/embed/ephy-web-view.c	2011-03-18 12:27:58.000000000 -0500
+++ src/epiphany-3.0.0/embed/ephy-web-view.c	2011-04-14 19:35:33.107544700 -0500
@@ -976,9 +976,8 @@ _ephy_web_view_hook_into_forms (EphyWebV
     /* We have a field that may be the user, and one for a password. */
     if (username_node && password_node) {
       LOG ("Hooking and pre-filling a form");
-      webkit_dom_event_target_add_event_listener (WEBKIT_DOM_EVENT_TARGET (form), "submit",
-                                                  G_CALLBACK (form_submitted_cb), false,
-                                                  web_view);
+      g_signal_connect (form, "submit-event",
+                        G_CALLBACK (form_submitted_cb), web_view);
 
       pre_fill_form (username_node, password_node, web_view);
 
